tests/test_timestream.py

```diff
@@ -345,3 +345,67 @@ def test_timestamp_measure_column(timestream_database_and_table):
         """,
     )
     assert df["measure_t"].dtype == "datetime64[ns]"
+
+
+def test_measure_name_multi(timestream_database_and_table):
```

Comment:
The two tests are mostly similar so perhaps we can consolidate with a boolean parameter and then have an if condition:
```
data = {"time": [datetime.now()] * 3}
args = {
    "database": timestream_database_and_table,
    "table=timestream_database_and_table,
    "time_col="time",
}

if multi:  # multi being the boolean parameter
    data.update({   
        "dim0": ["foo", "boo", "bar"],
        "dim1": [1, None, 3],
        "measure_0": [1.1, 1.2, 1.3],
        "measure_1": [2.1, 2.2, 2.3],
    })
    args.update(...)
else:
    ...
```